### DISCUSSION TITLE
Filter bills

### DISCUSSION THREAD

**Discussion by Gabrielsc** [firefly-iii/firefly-iii#8164] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

I need to no longer see bills that I no longer need to pay. I put an end date but it still shows as if I had to pay, even though there is a message on the screen saying, for example, attached "This bill ends in October", it pollutes the screen for November and the following months.
(Sorry for the English Google translate)
![image](https://github.com/firefly-iii/firefly-iii/assets/8128226/be46a881-16f0-47ca-9215-35df8d932996)

### Solution

If this was interesting, would you suggest something like filtering bills that are already expired or some way of archiving them. I wouldn't want to delete it so as not to lose the history.

**Comment by JC5** [comment]
Set them to inactive, add them to a group and sort the group to the bottom of the page.

**Reply by Gabrielsc** [reply]
The group's idea of even helping, yes, I will do more in my opinion would still not be ideal. I'm going to create a group for inactive people. Marking the bill as inactive blocks the display of whether the bill has been paid or not, showing a ~ symbol.
But if you don't intend to take the idea forward, you can mark the issue as closed.
![image](https://github.com/firefly-iii/firefly-iii/assets/8128226/b1582bc6-dfe5-4ced-a766-741877e51eb3)
